site/content/docs/5.2/components/buttons.md

```diff
@@ -210,7 +210,9 @@ Fancy larger or smaller buttons? Add `.btn-lg` or `.btn-sm` for additional sizes
 <button type="button" class="btn btn-secondary btn-sm">Small button</button>
 {{< /example >}}
 
-**It is not recommended by Orange Design System** but in some cases you could even roll your own custom sizing with CSS variables:
+{{< ods-incompatibility-alert >}}
+In some cases you could roll your own custom sizing with CSS variables like in the example just under this alert.
```

Comment:
To be more consistent with the other callout messages as written so far (see PR #1199), it would propose to write "This variant is not recommended by the Orange Design Specifications, nevertheless, in some cases, you could roll your own custom sizing with CSS variables."
Now, as there a topic to raise about the callout messages, it may be wiser to wait before considering changing this wording...